test/page.spec.ts

```diff
@@ -37,13 +37,17 @@ describe('Page', function () {
       const { context } = getTestState();
 
       const newPage = await context.newPage();
+      const evaluatePromise = newPage
+        .evaluate(() => new Promise(() => {}))
+        .catch((error_) => (error = error_));
```

Comment:
`error` is defined in line 48 so this will access an undefined variable.

Reply:
fixed